packages/cli/cloud/src/list-projects/command.ts

```diff
@@ -12,7 +12,7 @@ const command: StrapiCloudCommand = ({ command, ctx }) => {
     .description('List Strapi Cloud projects')
     .option('-d, --debug', 'Enable debugging mode with verbose logs')
     .option('-s, --silent', "Don't log anything")
-    .action(() => runAction('listProjects', action)(ctx));
+    .action(() => runAction('projects', action)(ctx));
```

Comment:
This is the name of the command that is displayed when there is an error message like when a user tries to run a command in a non strapi project folder: 'You need to run strapi <command> in a Strapi project...'